nvim/.config/nvim/lua/appearance.lua

```diff
@@ -0,0 +1,36 @@
+vim.cmd [[
+if exists('+termguicolors')
```

Comment:
check what this actually does...

Reply:
<img width="1126" alt="Screenshot 2022-11-05 at 8 44 02 AM" src="https://user-images.githubusercontent.com/42984735/200111532-7d84f920-4ee6-4d21-abe4-211c7e6e280d.png">
<img width="1112" alt="Screenshot 2022-11-05 at 8 43 32 AM" src="https://user-images.githubusercontent.com/42984735/200111534-320f594f-2055-4a0a-a5f6-012d2eb9f84b.png">

Seems that it keeps the background blue so will keep